docs/Using-Fleet/REST-API.md

```diff
@@ -5346,7 +5356,7 @@ Deletes the query specified by name.
 
 Deletes the query specified by ID.
 
-`DELETE /api/v1/fleet/queries/id/{id}`
```

Comment:
That is actually correct. The additional `/id/` is because, by default, queries are deleted by name. This endpoint is to indicate we want to delete the query by id.